### PR TITLE
Fix various flake8 warnings

### PR DESCRIPTION
**Pull request by consideRatio**
Extracted from #1381.

### CODE REVIEW

File: testing/local-binder-local-hub/binderhub_config.py [hunk 1]
```diff
@@ -1,14 +1,18 @@
...
-# `python -m binderhub -f binderhub_config.py`
-#
-# Override the external access URL for JupyterHub by setting the
-# environment variable JUPYTERHUB_EXTERNAL_URL
-
-# Host IP is needed in a few places
+"""
+A development config to test BinderHub locally.
+
+If you are running BidnerHub manually (not via JupyterHub) run
```
Comment (manics) [inline testing/local-binder-local-hub/binderhub_config.py:4]:
```suggestion
If you are running BinderHub manually (not via JupyterHub) run
```
Might as well fix my typo 😄

File: testing/local-binder-local-hub/binderhub_config.py [hunk 2]
```diff
@@ -1,14 +1,18 @@
...
+
+Override the external access URL for JupyterHub by setting the
+environment variable JUPYTERHUB_EXTERNAL_URL
+Host IP is needed in a few places
+"""
+import os
 import socket
+
+from binderhub.build_local import LocalRepo2dockerBuild
+
```
Comment (manics) [inline testing/local-binder-local-hub/binderhub_config.py:15]: Moving imports to the top is fine, but the old imports are still present on lines 17-18.

File: binderhub/tests/test_registry.py [hunk 3]
```diff
@@ -69,7 +69,7 @@ def initialize(self, test_handle):
         self.test_handle = test_handle
 
     def get(self):
-        scope = self.get_argument("scope")
+        _ = self.get_argument("scope")
```
Comment (manics) [inline binderhub/tests/test_registry.py:72]: Since the result isn't needed maybe the whole line can be deleted? Not a big deal to keep it though.

**Review by manics — COMMENTED** [review]
Some duplicate imports, otherwise LGTM

**Comment by consideRatio** [comment]
Thanks for the quick review @manics! I solved a merge conflict by rebasing and addressed all your comments!

**Review by manics — APPROVED** [review]

**Pull request merged**